app/services/upload_sanitizer.rb

```diff
@@ -0,0 +1,18 @@
+# Formats uploaded files to UTF-8 encoding and strips unexpected BOM characters.
+# Takes an open File object as input
+class UploadSanitizer
```

Comment:
here we took option 1 for the name of this service as described here:
https://community.openfoodnetwork.org/t/how-to-name-a-new-service-in-ofn/1530

we should make a decision and stick to some naming convention, otherwise, the service naming chaos will keep increasing.
@Matt-Yorkley @sauloperez @kristinalim could you guys please vote on your favorite options in the discourse thread above? thanks!